Source/Dafny/Resolver.cs

```diff
@@ -3489,161 +3499,126 @@ public class ModuleBindings
           }
         } else if (e is ConversionExpr ce) {
           Object o = GetAnyConst(ce.E, consts);
```

Comment:
The type `System.Object` is more typically written as `object` in C#.

Reply:
Fixed